packages/xod-client-electron/test-func/suite.spec.js

```diff
@@ -177,5 +177,18 @@ describe('IDE', () => {
         .then(() => page.getCodeboxValue())
         .then(code => assert.strictEqual(code, expectedCpp, 'Actual and expected C++ don’t match'))
     );
+    it('closes show code popup', () =>
+      page.closePopup()
+    );
+  });
+
+  describe('deleting a patch', () => {
+    it('opens patch from xod/core', () =>
+      page.openPatchFromProjectBrowser('clock')
+        .then(() => page.findPatchGroup('welcome-to-xod').click())
+        .then(() => page.assertPatchGroupExpanded('welcome-to-xod'))
+        .then(() => page.deletePatch('my-blink'))
+        .then(() => page.assertNodeUnavailableInProjectBrowser('my-blink'))
+    );
```

Comment:
Misguiding describe/it text. Now it is “deleting a patch opens patch from xod/core” :thinking: Does not reflect what’s going on. Why do we ever open the `clock`? Is it because we can’t delete a patch of the active tab? If so, lets add a comment about it.

Splitting into two its (one about open, another about deletion) would also make things more clear.